lib/dams/controller_helper.rb

```diff
@@ -44,6 +44,71 @@ def osf_contributors(document)
 		  osf_data = (osf_data.blank?) ? osf_data << {"name": "UC San Diego Library"} : osf_data
 		end
 
+		def osf_related_agents(document)
```

Comment:
Could you check the spacing on this file? It seems off/not aligned.

Reply:
@mcritchlow The spacing for new added code looks fine to me from my local sublime editor which are all two space indent. Some old code such as #get_linked_object_label() and etc. are not aligned in this file. Do you want me to clean up those old code?

![screen shot 2018-06-04 at 12 12 03 pm](https://user-images.githubusercontent.com/2432753/40936603-b38476f0-67f0-11e8-92ab-f00757c31a0a.png)

![screen shot 2018-06-04 at 12 13 15 pm](https://user-images.githubusercontent.com/2432753/40936623-c0c601ee-67f0-11e8-959d-2dfffa0d2a86.png)

Reply:
Ah you're right. Those were getting mixed up in the diff. But your OSF-related methods do indeed look fine after I looked at them in a non-diff context. Sorry about that :)

Reply:
Or maybe it is better to create a separate ticket to clean up and format old code if it is necessary?

Reply:
Personally I don't think it's the best use of your time. I think if/when we have to touch that code, we could use the opportunity to clean it up a bit. Just my opinion though :)

Reply:
@mcritchlow  I found the reason why in the editor my code looks beautiful, but it looks far off in Github. That's because Github displays tabs as equivalent to 8 spaces. Now I update my editor setting to make the code in Sublime and Github look the same.  I can see the off space and will fix them from my editor. Sorry about that.

Reply:
Oh no worries. That actually was my original concern, that IDE "hide" spacing issues so one would never actually know that they exist. My editor (Vim) definitely does not do that ;)

Though this does make me wonder if we should write down what our expectations are for tabs/spaces and the spacing for them. Rubocop kind of enforces it, but we've also never written it down and we more or less are barely using Rubocop w/ damspas because it's got so much historical baggage. One thing I came across that I was going to run by you all is https://editorconfig.org/ which works with pretty much every text editor/IDE. Maybe I should ask in our slack channel?

Reply:
Yes, sounds a good topic to discuss.